CoverageProfiler/CoverageProfiler.inputs.json

```diff
@@ -0,0 +1,32 @@
+{
+  "coverageProfile.referenceFasta": "File",
+  "coverageProfile.SamtoolsDepth.samtools_docker": "String (optional, default = \"euformatics/samtools:1.20\")",
```

Comment:
I'm a little uncomfortable using dockers from random companies.  What else do we depend on in this docker, if it is just samtools 1.2 we should be able to find a more standard source.

Reply:
This is samtools 1.20 which offers the fragment count for depth. I don't see any other places offer the latest Samtools docker besides that company. I could build it myself and put it in TAG GAR repo. But, I would prefer to use a pre-built docker.

Reply:
@fleharty Here are two alternative options with the latest samtools docker:
1. [staphb/samtools:1.20](https://hub.docker.com/r/staphb/samtools/tags)
Stored on Dockerhub from State Public Health Bioinformatics Community
 2. [quay.io/biocontainers/samtools:1.20--h50ea8bc_0](http://quay.io/biocontainers/samtools:1.20--h50ea8bc_0)
 Stored on quay.io From BioContainer
 
Which one would you prefer to be default of samtools docker in this WDL? It has to be 1.20 to fully take advantage of fragment count.

Reply:
I would prefer we use quay.io